exposee-app/server.js

```diff
@@ -65,7 +113,21 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
-
+app.post('/broadcast', async (req, res)=> {
+  const userId = req.body.id;
+  try{
```

Comment:
I think you need to run the eslint to get rid of these formatting inconsistencies.